packages/opc-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@one-platform/opc-base",
-  "version": "1.0-beta",
+  "version": "1.0.0-beta",
```

Comment:
@akhilmhdh after changing the version you should run `npm i` once again so that even the package-lock is updated

Reply:
Done

Reply:
@akhilmhdh I don't see the package-lock.json change in these changes

Reply:
Oopz forgot the push command 😄